front/app/containers/ProjectsShowPage/messages.ts

```diff
@@ -399,4 +399,13 @@ export default defineMessages({
     id: 'app.containers.Projects.a11y_phase',
     defaultMessage: 'Phase {phaseNumber}: {phaseTitle}',
   },
+  emailSharingSubject: {
+    id: 'app.containers.Projects.emailSharingSubject',
+    defaultMessage: 'Join the project: {projectName}.',
+  },
+  emailSharingBody: {
+    id: 'app.containers.Projects.emailSharingBody',
+    defaultMessage:
+      'What do you think of this project? Join the discussion at {projectUrl} to make your voice heard!',
+  },
```

Comment:
I've added these messages in for now, but someone in Product should probably change them.

Reply:
Definitely! Ask for Ben to review the translations on the branch?